### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by mansipadgelwar**
Add Sign-up, log in, and Logout page.
Screenshot 2022-03-20 at 4 18 36 PM
Screenshot 2022-03-20 at 4 17 44 PM
Screenshot 2022-03-20 at 4 16 29 PM

### CODE REVIEW

File: authentication/auth.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+.input-icon > input{
+    max-width: 21.88rem;
+}
+
+.icons-right{
+    position: absolute;
+    right: 1.56rem;
+    bottom: 1.43rem;
```
Comment (samyakshah3008) [inline authentication/auth.css:37]: Minor suggestion: You can use shorthand property here for less lines of code, can refactor later

File: authentication/auth.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    border: none;
+    background-color: var(--white);
+}
+
+.form{    
+    display: flex;
+    flex-direction: column;    
+    align-content: center;    
+    flex-wrap: wrap;
+    width: 25rem;
```
Comment (samyakshah3008) [inline authentication/auth.css]: Can give max or min-width for responsiveness

**Review by samyakshah3008 — APPROVED** [review]
Liked the color theme 
Clean UI 
Good naming conventions, they are self explanatory 

Given few minor suggestion in single comments, can add later on 

Rest is all good for approval ✅

**Review by shrey27 — APPROVED** [review]
code structure is fine
PR is good to merge.

**Pull request merged**